src/components/HamburgerMenu.tsx

```diff
@@ -37,19 +38,27 @@ const EditDeleteMenu = styled.div`
 const HamburgerMenu = (props: HamburgerMenuProps) => {
   const [isMenuOpened, setIsMenuOpened] = useState(false);
 
+  const onMenuItemClick = (type: ModalWindowType) => {
+    props.selectMovie();
+    props.openModalWindow(type);
+  };
+
   return (
-    <HamburgerMenuWrapper onClick={() => setIsMenuOpened(!isMenuOpened)}>
+    <HamburgerMenuWrapper
+      onClick={(event: any) => {
```

Comment:
type is `React.MouseEvent<HTMLDivElement> `

Reply:
Fixed.